### PR TITLE
(procedures) Upload files

### PR DESCRIPTION
**Pull request by Crash--**
A simple implementation of the upload.

### CODE REVIEW

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx [hunk 1]
```diff
@@ -1,16 +1,66 @@
...
-  </Button>
+import { withClient } from 'cozy-client'
+import {
+  translate,
+  ButtonLink,
+  Icon,
+  FileInput
+} from 'cozy-ui/transpiled/react/'
+
+import creditApplicationTemplate from '../../templates/creditApplicationTemplate'
```
Comment (y-lohse) [inline packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: Nope, please use the new import path.

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx [hunk 2]
```diff
@@ -1,16 +1,66 @@
...
+import DocumentsDataFormContainer from '../../containers/DocumentsDataForm'
+const EmptyDocumentHolder = ({
+  t,
+  documentId,
+  client,
+  linkDocumentSuccess
+}) => (
+  <FileInput
+    onChange={async file => {
+      const dirPath = creditApplicationTemplate.pathToSave
```
Comment (y-lohse) [inline packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: Would be cool to move this in a method.

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx (hunk 1 above)
Comment (Crash--) [reply packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: Woa ! cherrypick. Good catch thanks !

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx [hunk 3]
```diff
@@ -1,16 +1,66 @@
...
+        }
+        const dirId = await filesCollection.ensureDirectoryExists(dirPath)
+
+        const createdFile = await client
+          .collection('io.cozy.files')
+          .createFile(file, { dirId, metadata })
+
+        linkDocumentSuccess({ document: createdFile.data, documentId })
+      } catch (uploadError) {
+        console.log('errr', uploadError)
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: `console.error`?

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx (hunk 2 above)
Comment (cedricmessiant) [reply packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: are we sure we want to upload the docs and create the zip at the same path? Maybe it's ok for now but we could use a `pathToSaveZip` or something

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx (hunk 3 above)
Comment (Crash--) [reply packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: It'll be removed in the next PR with conflicts ;)

File: packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx (hunk 2 above)
Comment (Crash--) [reply packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx]: ATM we're not sure where we want to save the files and the content. So let's keep it that way and see what happen next, after a real test :)

**Comment by Crash--** [comment]
thanks for the suggestion @y-lohse. I've a more work to do for the refacto :D

**Pull request merged**